lib/index.js

```diff
@@ -2210,6 +2210,46 @@ addMethod('flatMap', function (f) {
     return this.map(f).sequence();
 });
 
+/**
+ * Creates a new Stream of values by applying each item in a Stream to each
+ * value in a Stream of functions
+ *
+ * @id ap
+ * @section Higher-order Streams
+ * @name Stream.ap(m)
+ * @param {Stream} m - incoming stream of function(s) to apply to value(s) in stream
+ * @api public
+ *
+ * var filenames = _(['foo', 'bar', 'baz']);
```

Comment:
Can you come up with a better example code? One with multiple functions being applied to a value stream, similar to what you have in the `reflect timing of value and function arrival` test.

This example doesn't really show why you would use `ap` over `map`.